lib/python/qmk/cli/cd.py

```diff
@@ -0,0 +1,14 @@
+"""Open a shell in the QMK Home directory
+"""
+import pty
```

Comment:
seems a no go on msys
```
>>> import pty
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "D:/QMK_MSYS/mingw64/lib/python3.8/pty.py", line 12, in <module>
    import tty
  File "D:/QMK_MSYS/mingw64/lib/python3.8/tty.py", line 5, in <module>
    from termios import *
ModuleNotFoundError: No module named 'termios'
```

Reply:
Can confirm this error

Reply:
I think it should work now on Windows as well.